src/app/public/modules/flyout/flyout.component.ts

```diff
@@ -183,10 +183,15 @@ export class SkyFlyoutComponent implements OnDestroy, OnInit {
   public onWindowResize(event: any): void {
     if (this.flyoutMediaQueryService.isWidthWithinBreakpiont(event.target.innerWidth,
       SkyMediaBreakpoints.xs)) {
-        this.updateBreakpointAndResponsiveClass(event.target.innerWidth);
+      this.updateBreakpointAndResponsiveClass(event.target.innerWidth);
     } else {
       this.updateBreakpointAndResponsiveClass(this.flyoutWidth);
     }
+
+    if (event.target.innerWidth - this.flyoutWidth < 20) {
```

Comment:
Since `20` is replicated several times, what if we made it a constant?

Reply:
Done